src/main/java/catan/services/util/play/CardUtil.java

```diff
@@ -0,0 +1,52 @@
+package catan.services.util.play;
+
+import catan.domain.exception.PlayException;
+import catan.domain.model.game.DevelopmentCards;
+import catan.domain.model.game.types.DevelopmentCard;
+import catan.services.util.random.RandomUtil;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Component
+public class CardUtil {
+    private Logger log = LoggerFactory.getLogger(CardUtil.class);
+
+    public static final String CARDS_ARE_OVER_ERROR = "CARDS_ARE_OVER";
+
+    private RandomUtil randomUtil;
+
+    public DevelopmentCard pullDevelopmentCard(DevelopmentCards availableDevelopmentCards) throws PlayException {
+        List<DevelopmentCard> availableDevelopmentCardsList = formAvailableDevCardsList(availableDevelopmentCards);
+        validateThereAreAvailableCards(availableDevelopmentCardsList.size());
+
+        return randomUtil.pullRandomDevelopmentCard(availableDevelopmentCardsList);
+    }
+
+    private void validateThereAreAvailableCards(int availableDevelopmentCardsQuantity) throws PlayException {
```

Comment:
since you create separate method, I would pass the whole list, and get size() of list directly here